src/briefcase/commands/create.py

```diff
@@ -16,12 +16,18 @@
 from briefcase.integrations import git
 from briefcase.integrations.subprocess import NativeAppContext
 
-from .base import (
-    BaseCommand,
-    TemplateUnsupportedVersion,
-    UnsupportedPlatform,
-    full_options,
-)
+from .base import BaseCommand, TemplateUnsupportedVersion, full_options
+
+
+class UnsupportedPlatform(BriefcaseCommandError):
+    def __init__(self, platform):
+        self.platform = platform
+        super().__init__(
+            f"""\
+App cannot be deployed on {platform}. This is probably because one or more
+requirements (e.g., the GUI library) doesn't support {platform}.
+"""
+        )
```

Comment:
It's not clear why this exception is being moved to `create`; but If we're going to move this exception, I'd prefer to see it move to `briefcase.exceptions`.

Reply:
I moved it because its use moved from `base` to `create` a little while ago. There are several Exceptions, though, outside of `briefcase.exceptions`; I can move all of them there.